tests/README.md

```diff
@@ -0,0 +1,22 @@
+# Intel® Media SDK Open Source TEst Driver
+
+### This documentation is created to run tests on Linux OS distributions
+
+#### Prerequisites:
+Before running tests ensure that required packages are installed on your system:
+```
+git
+git-lfs
+python >= 3.6
+```
+Media stack ingredients (libva, libmfx, libigdgmm, libigfxcmrt) should be installed to `/opt/intel/mediasdk`
+
+Media driver should be compiled and installed to `/opt/intel/msdk_driver`
+
+#### Run tests:
+```sh
+export LD_LIBRARY_PATH=/opt/intel/mediasdk/lib64
```

Comment:
Here is how I would imagine run of the `ted.py` in most generic case:
```
export PREFIX=/path/to/something
export BINDIR=$PREFIX/bin
export LIBDIR=$PREFIX/lib

cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=$LIBDIR ..
make && make install

export PATH=$BINDIR:$PATH
export LD_LIBRARY_PATH=$LIBDIR

python3 ted.py --gold  # to collect reference results
python3 ted.py
```

I don't quite get why do we need to runs of ted with and without --gold, but ok for now.